src/linear_algebra/determinant.lean

```diff
@@ -368,6 +368,16 @@ end
 lemma basis.is_unit_det (e' : basis ι R M) : is_unit (e.det e') :=
 (is_basis_iff_det e).mp ⟨e'.linear_independent, e'.span_eq⟩
 
+/-- Any alternating map to `R` where `ι` has the cardinality of a basis equals the determinant
+map with respect to that basis, multiplied by the value of that alternating map on that basis. -/
+lemma alternating_map.eq_smul_det (f : alternating_map R M R ι) : f = f e • e.det :=
```

Comment:
```suggestion
lemma alternating_map.eq_smul_basis_det (f : alternating_map R M R ι) : f = f e • e.det :=
```
since there's no mention of `basis` in the name otherwise?